canonical/canonical.com
# Implement the Jinja logic for the desktop navigations dropdowns

## Done

- Implements the logic for rendering the 3 different dropdown views, [see figma](https://www.figma.com/design/p0y7lHJ7edLIeSXM1qbpRZ/23.10-C.com-%26-U.com---Navigation?node-id=455-9266&t=0kWDK2TbW11Y4Z0N-0)
- Makes some changes to the navigaiton.yaml to make processing it easier
- This led to some refactoring of how the mobile view is rendered
- Some refactoring to reuse some templates and move sections into their own templates
- move the SCSS into a separate file with separate mixins. This is mostly to make up-streaming it easier

## QA
**please ignore keyboard navigation, careers roles not appearing and the giant black boxes which will later be images, they are handled in separate tickets**

- Compare what I have to [the figma design](https://www.figma.com/design/p0y7lHJ7edLIeSXM1qbpRZ/23.10-C.com-%26-U.com---Navigation?node-id=455-9266&t=0kWDK2TbW11Y4Z0N-0)
- Make sure all the content is there and the links work etc
- Check the Jinja logic and Scss makes sense etc, check for repeating code or when things could be made clearer (naming, structure etc)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12670

## Screenshots

View 1:
![image](https://github.com/user-attachments/assets/2dae9214-6882-4f4b-9d04-f1e8ed96751a)

View 2:
![image](https://github.com/user-attachments/assets/775fa0f8-e9b5-4fac-9f63-655d201d2d89)

View 3:
![image](https://github.com/user-attachments/assets/354f2f8c-f0e7-4b82-8cd5-4de1d7f55010)
